src/algebraic_topology/simplicial_object.lean

```diff
@@ -26,6 +31,15 @@ This is the category of contravariant functors from `simplex_category` to `C`. -
 def simplicial_object := simplex_categoryᵒᵖ ⥤ C
 
 namespace simplicial_object
+
+instance [has_limits C] : has_limits (simplicial_object C) :=
+let E : (simplex_categoryᵒᵖ ⥤ C) ≌ (ulift.{v} simplex_category)ᵒᵖ ⥤ C :=
+  ulift.equivalence.op.congr_left in adjunction.has_limits_of_equivalence E.functor
```

Comment:
Could I suggest newlines after the `in`, for readability?

Reply:
Done!